tools/generate-bls-tests.py

```diff
@@ -28,23 +28,26 @@ def print_validation_test_case(f1, f2, num_cases, filter_pk, filter_msg, filter_
     cost = 4999087
     messages = []
     sigs = []
-    f1.write("bls_verify (")
```

Comment:
Curiously, this file makes test-blspy-verify.txt and test-blspy-pairing.txt a bit redundant since it can generate them. That means technically we could get away with not checking in the .txt files since they could just be generated as-needed by the CI or tests. I don't think we need to make this change at this time, but it's an interesting thing to think about.

Reply:
yeah. I'm tempted to make the CI ensure the generated test cases are up-to-date. It's convenient to be able to run `cargo test` locally without having to generate these files too.